src/msgraph_core/middleware/telemetry.py

```diff
@@ -52,22 +65,29 @@ def _append_sdk_version_header(self, request) -> None:
         version of the client SDK library(s).
         Also adds the featureUsage value.
         """
+        sdk_name = 'graph-python-core'
+
+        if self.options.api_version == APIVersion.v1:
+            sdk_name = 'graph-python'
+        if self.options.api_version == APIVersion.beta:
+            sdk_name = 'graph-python-beta'
+
         if 'sdkVersion' in request.headers:
             sdk_version = request.headers.get('sdkVersion')
-            if not sdk_version == f'graph-python-core/{SDK_VERSION} '\
+            if not sdk_version == f'{sdk_name}/{SDK_VERSION} '\
                 f'(featureUsage={request.context.feature_usage})':
                 request.headers.update(
                     {
                         'sdkVersion':
-                        f'graph-python-core/{SDK_VERSION},{ sdk_version} '\
+                        f'{sdk_name}/{SDK_VERSION},{ sdk_version} '\
                         f'(featureUsage={request.context.feature_usage})'
                     }
                 )
         else:
             request.headers.update(
                 {
                     'sdkVersion':
-                    f'graph-python-core/{SDK_VERSION} '\
+                    f'{sdk_name}/{SDK_VERSION} '\
```

Comment:
From the [SDK design examples](https://github.com/microsoftgraph/msgraph-sdk-design/blob/master/middleware/TelemetryHandler.md), when the service lib is used, we also need to append the Core version in use

Reply:
Good catch, I wasn't aware of that. Thanks!

Reply:
You might have forgotten to make the change :)